Integrations/src/main/java/io/deephaven/integrations/python/PythonTools.java

```diff
@@ -0,0 +1,36 @@
+package io.deephaven.integrations.python;
+
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Functions to support interfacing to python from java
+ */
+public class PythonTools {
+    /**
+     * Convert a map to a function.
+     *
+     * @param map           the map to convert to function; the caller gives up ownership to the returned object
+     * @param defaultValue  the value to map to any keys not in the map; may be null
+     * @return              the resulting function
+     */
+    @SuppressWarnings("unused")
+    public static Function<String, String> functionStrStrFromMap(
+            final Map<String, String> map,
+            final String defaultValue
+    ) {
+        return (final String key) -> map.getOrDefault(key, defaultValue);
+    }
+
+    /**
+     * Convert a map to a function, mapping any nonexistent keys in the map to themselves.
+     *
+     * @param map           the map to convert to function; the caller gives up ownership to the returned object
+     * @return              the resulting function
+     */
+    @SuppressWarnings("unused")
+    public static Function<String, String> functionStrStrFromMap(final Map<String, String> map
```

Comment:
I think the overload is not quite clear ; the identity behavior is not exactly normal; so I would add Identity or something to the function name to make it less ambiguous.

Reply:
Done.